docs/tutorials/pizza-party.md

```diff
@@ -84,23 +85,27 @@ into **the end** of the<br/>
 ``||loops(noclick): on start||`` <br/>
 container already in the workspace.
 
-- :paint brush: Click the empty square in the background block to choose an image.
-![Switch to the My Assets tab](/static/skillmap/assets/my-assets-three.png " ")
-Switch to the **My Assets** window to select the **cityscape** background.
+- :paint brush: Click the empty square in the background block and flip to **Gallery**
+![Switch to the My Assets tab](/static/skillmap/assets/gallery.png " ")
```

Comment:
Should this say Gallery tab?

Reply:
It could be "flip to **Gallery**" or "flip to the **Gallery** tab".